### PR TITLE
Add '--add-opens' for reflection

### PR DESCRIPTION
**Pull request by abudevich**

### CODE REVIEW

File: 2.0/gradle/vividus-test.gradle [hunk 1]
```diff
@@ -129,9 +129,12 @@ class StoriesExec extends VividusJavaExec {
     String fileToSaveExitCode
 }
 
+tasks.withType(VividusJavaExec).configureEach {
+    jvmArgs += defaultJvmArgs
+}
```
Comment (valfirst) [inline 2.0/gradle/vividus-test.gradle:134]: why is this change needed?

Comment (abudevich) [reply 2.0/gradle/vividus-test.gradle:134]: because of this flag needed not only for `StoriesExec` tasks.
For example, I caught  exception on `testVividusInitialization` and `validateKnownIssues` tasks

**Comment by valfirst** [comment]
This solution introduces too much overhead, we'll go with jitpack.
